### ISSUE TITLE
Temporary remove dulcy from rotation

### ISSUE THREAD

**Issue by mtmail** [openstreetmap/operations#920]
dulcy serves a pre-release of Nominatim version 5 (python rewrite). It introduced changes that cause issues for users of the service. See https://github.com/osm-search/Nominatim/issues/3134#issuecomment-1665769426

Sarah is on leave at the moment. Grant suggested in https://github.com/openstreetmap/openstreetmap-website/issues/4137 that the server can be removed if needed.

Unless there’s other communication with Sarah I‘d like to ask to remove it temporarily, it seems to break too many user‘s websites.

**Comment by tomhughes** [comment]
Do you have any evidence to support your claim?

Most of the reports I've seen are for sites that are sending queries that are at least arguably bogus. I have no evidence of widespread major breakage significant enough to warrant taking the machine our of service and I'd rather continue to gather details of problems so that they can be fixed if necessary.

**Comment by gravitystorm** [comment]
> Do you have any evidence to support your claim?

Wow, that's not a helpful response! 

@mtmail linked to two different places that describe the problems people are having with the version of nominatim running on dulcy - first, an issue with many comments from users of nominatim, and secondly, an issue showing breakage on openstreetmap.org itself! It's hard to understand how that's not evidence of a problem.

https://github.com/osm-search/Nominatim/issues/3138 is the main thing here for me - the response format has (inadvertently?) changed, @mtmail caught it causing a problem for osm.org, and so I think it's reasonable to suggest something needs to be done.

Now maybe I'm misunderstanding what's going on, and maybe @mtmail is too, and maybe you've already taken steps to avoid this from impacting the osm.org frontpage - but in that case, it would be nice to told what we're misunderstanding, rather than getting a curt brush-off.

**Comment by tomhughes** [comment]
Well the first one of those is a classic example of the kind of thing I'm talking about, where it's specific to clients that are doing something a bit odd in adding a trailing slash to the endpoint.

The second one is probably the only thing I'm aware of that might lead me to disable it. I did consider adding a workaround to the rails code this morning but decided I couldn't really make a case for it at least until we get a decision from Nominatim about what the correct response format is. Equally I decided that given that server only takes a relatively small proportion of requests and we're only talking about a couple of days that it wasn't worth disabling it.
